content/soda-gettingstarted/installation-using-ansible.md

```diff
@@ -5,94 +5,47 @@ weight: 40
 ---
 This document describes how to install SODA projects in a local cluster with detailed configuration options. These steps will help you to enable / disable projects. After installation using these steps, you can get the features of all the enabled projects. You can test either through SODA Dashboard UI or CLI
 
-`Hereafter, Hotpot refers to API, Controller and Dock projects`
 
 {{% notice info %}}
-**Ubuntu 16.04 support for SODA is deprecated as of SODA Jerba release v1.4.0.** <br />
-**If you have a requirement to install SODA on Ubuntu 16.04, please contact us on slack and we will try to help with the setup.**
-{{% /notice %}}
-
-{{% notice info %}}
-**Ubuntu 20.04 support for SODA is added as experimental in Lamu v1.6.0 release through branch `ubuntu2004-experimental`.** <br />
-**If you have a requirement to install SODA on Ubuntu 20.04, please [refer](https://github.com/sodafoundation/installer/tree/ubuntu2004-experimental) or contact us on slack and we will try to help with the setup.**
+**Below process of installation will support for Ubuntu 20.04 & Ubuntu 18.04.** <br />
 {{% /notice %}}
 
 ---
 
 ### Pre-requisites
-- SODA installation is tested on `Ubuntu 18.04`. 
-- `root` user is REQUIRED before the installation work starts.
-
-#### Install following packages:
-
-```bash
-apt-get update && apt-get install -y git make curl wget libltdl7 libseccomp2 libffi-dev gawk
-```
----
-
-#### Install docker:
-
-
-##### For Ubuntu 18.04
-
-```bash
-wget https://download.docker.com/linux/ubuntu/dists/bionic/pool/stable/amd64/docker-ce_18.06.1~ce~3-0~ubuntu_amd64.deb
-```
+- SODA installation is tested on `Ubuntu 18.04`& `Ubuntu 20.04`. 
+- Python 3.6 or above should be installed
 
 ```bash
-dpkg -i docker-ce_18.06.1~ce~3-0~ubuntu_amd64.deb
+sudo apt-get update && sudo apt-get install -y git
 ```
----
-####  Install docker-compose:
+- Ensure no ansible & docker installed, OR Lastest ansible and docker tools are installed with versions listed below or later. If ansible & docker is not installed in the OS, script          install_dependencies.sh will install it.
```

Comment:
```suggestion
- Ensure no ansible & docker installed, OR Lastest ansible and docker tools are installed with versions listed below or later. If ansible & docker is not installed in the OS, script          `install_dependencies.sh` will install it.
```

Reply:
If ansible & docker is not installed in the OS -> If ansible & docker **are** not installed in the OS